### ISSUE TITLE
Edit content field and remove type field from designer-life.yml credits file

### ISSUE THREAD

**Issue by blulady** [hackforla/website#2814]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
As a developer, I want to edit the content field and remove the type field from a credit's yml file so that redundant code is removed and the code is easier to understand.

### Details
Currently, in each credit's yml file, there is a `content` field and a `type` field, which contains similar information. Since the fields are redundant, we decided to remove the `type` field.  

Also, we changed the `content` field to the `content-type` field to make it clearer what its purpose is. The possible values for `content-type` field are image, video, or audio; other types of content can be added if necessary. In the future, this will allow developers a way to differentiate easily between different types of content in order to show each credit's media file correctly on the website.

### Action Items
For the file `_data/internal/credits/designer-life.yml`, do the following:
- [x] Change line 4 for the content field from
  ```
  content: icon
  ```
  to
  ```
  content-type: image
  ```
- [x] Remove line 11 for the type field
  ```
  type: icon
  ```
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #2775 by
  - [x] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #2775 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is tracked in the epic #2775.

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `_data/internal/credits/designer-life.yml`

- Click on the links below to see the source file code:
https://github.com/hackforla/website/blob/e980b70efb40f63c7c1d54595c0bc3b1ab0d3569/_data/internal/credits/designer-life.yml#L4
https://github.com/hackforla/website/blob/e980b70efb40f63c7c1d54595c0bc3b1ab0d3569/_data/internal/credits/designer-life.yml#L11

**Comment by blulady** [comment]
I cannot figure out my blocker. I have followed the instructions on the contributing.md to the point of making a git push, I then get this error:
```C:\Users\ssang\Documents\hackforla\website>git push --set-upstream origin gh-pages
To https://github.com/blulady/website.git
 ! [rejected]        gh-pages -> gh-pages (non-fast-forward)
error: failed to push some refs to 'https://github.com/blulady/website.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.```

Please Advise

**Comment by SAUMILDHANKAR** [comment]
> I cannot figure out my blocker. I have followed the instructions on the contributing.md to the point of making a git push, I then get this error:
> 
> ```
> To https://github.com/blulady/website.git
>  ! [rejected]        gh-pages -> gh-pages (non-fast-forward)
> error: failed to push some refs to 'https://github.com/blulady/website.git'
> hint: Updates were rejected because the tip of your current branch is behind
> hint: its remote counterpart. Integrate the remote changes (e.g.
> hint: 'git pull ...') before pushing again.
> hint: See the 'Note about fast-forwards' in 'git push --help' for details.```
> 
> Please Advise
> ```

@blulady Is your blocker related to #2772. If yes, I would recommend marking your comment unrelated and hiding it from this issue and posting your blocker there. Thanks.

**Comment by JohnJBarrett22** [comment]
i. Availability: (When are you available to work on the issue/answer questions other programmers might have about your issue?) - I am have open availability, minus doctor appointments for my Wife that are not consistent. Consider me open.
ii. ETA: (When do you expect this issue to be completed?) The issue will be completed by the end of the night on 8/10/22.
